app/questionPage.html

```diff
@@ -46,16 +49,36 @@ <h3 id="timer"> 30m 00s </h3>
             </div>
             <div class="question-container col-md-8">
                 <div id="questions"></div>
+                <div id="question-nav" class="question-nav"></div>
                 <button type="button" id="finish" class="btn btn-success finish">Finish</button>
             </div>
-            <div class="col-md-2">
+            <div class="col-md-2 overview_column">
                 <button type="button" class="btn btn-info next">Next</button>
+                <button type="button" class="btn btn-info overview">Overview</button>
             </div>
             <div class="col-md-2">
             </div>
         </div>
     </div>
 </div>
+<div id="overview_page">
+    <h1>Test Overview</h1>
+    <table>
+        <thead>
+            <tr>
+                <th>Question Id</th>
+                <th>Question</th>
+                <th>Answered</th>
+                <th>Flagged</th>
+            </tr>
+        </thead>
+        <tbody class="overview_table_body"></tbody>
+    </table>
+    <footer class="overview_footer">
+        <p>Clicking this button submits the test once and for all there is no turning back!</p>
+        <button type="button" id="finish" class="btn btn-success finish">Finish</button>
```

Comment:
duplicate id 'finish'

Reply:
Used the same id because we'll get rid of the one inside question_page later on.